app/helpers/poem_helper.rb

```diff
@@ -1,8 +1,20 @@
 module PoemHelper
+  # ToDo 他の箇所でも使うようになったら(コメントとか)
```

Comment:
__[rubocop]__
* Use only ascii symbols in comments. [:ref](https://github.com/bbatsov/ruby-style-guide#english-comments)
* Annotation keywords like `ToDo` should be all upper case, followed by a colon, and a space, then a note describing the problem. [:ref](https://github.com/bbatsov/ruby-style-guide#annotate-keywords)